webapp/src/main/webapp/WEB-INF/conf/officeonline/portal/intranet/navigation.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+
+    Copyright (C) 2019 eXo Platform SAS.
+
+    This is free software; you can redistribute it and/or modify it
+    under the terms of the GNU Lesser General Public License as
+    published by the Free Software Foundation; either version 2.1 of
+    the License, or (at your option) any later version.
+
+    This software is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public
+    License along with this software; if not, write to the Free
+    Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+    02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+-->
+<node-navigation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_3 http://www.gatein.org/xml/ns/gatein_objects_1_3"
+  xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_3">
+  <priority>1</priority>
+  <page-nodes>
+    <node>
+      <uri>oeditor</uri>
+      <name>oeditor</name>
```

Comment:
Let's use "mseditor" for the moment to avoid conflicts with another editor addon.

Reply:
Fixed in https://github.com/exo-addons/office-online/pull/4/commits/02ae41be8f07d7ad2cc22669394cc34855d37e52